.github/workflows/test-javascript.yml

```
@@ -0,0 +1,31 @@
+name: 'test reports javascript'
+
+on:
+  push:
+    branches:
+      - "main"
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: prefix-dev/setup-pixi@v0.8.1
+        with:
+          pixi-version: v0.24.0
+          environments: ci-py312-latest-optional-deps
+          # we can freeze the environment and manually bump the dependencies to the
+          # latest version time to time.
+          frozen: true
+
+      - name: Create the reports
+        working-directory: skrub/_reporting/js_tests/
+        run: pixi run make_reports
```

For consistency, could you use hyphen instead of underscore.

```suggestion
        run: pixi run make-reports
```